keyboards/split75/README.md

```diff
@@ -0,0 +1,26 @@
+# split75
+
+![split75](https://i1.wp.com/mechboards.co.uk/wp-content/uploads/2019/10/HTB1qf0aSOLaK1RjSZFxq6ymPFXae.jpg)
+
+This is a port of the QMK firmware for the split75 keyboard, also known as the ve.a taobao split clone. It runs on
+ps2avrGB firmware, like the [ps2avrGB keyboard](https://www.keyclack.com/product/gb-ps2avrgb/) or the ones sold
+by [Winkeyless](http://winkeyless.kr/product/ps2avrgb-parts/).
+
+Original work performed by Michael L. Walker, all credit goes to him. I've tidied up the keymap (removed 3 extra keys), a couple of functions, info.json and this Readme.
+
+* Keyboard Maintainer: [Michael L. Walker](https://github.com/walkerstop)
+* Hardware Availability: https://a.aliexpress.com/_BT9raf https://item.taobao.com/item.htm?id=556970472669&toSite=main
+
+Make example for this keyboard (after setting up your build environment):
+
+    make split75:default
+
+Flashing example for this keyboard ([after setting up the bootloadHID flashing environment](https://docs.qmk.fm/#/flashing_bootloadhid))
+
+    make split75:default:flash
+	or
+	.\bootloadHID -r PATH_TO_FOLDER\split75_default.hex
```

Comment:
Given the binary was removed in another commit, we could chose to loose this block or update it to assume bootloadHID cli is installed globally.

```suggestion
	bootloadHID -r PATH_TO_FOLDER\split75_default.hex
```

Reply:
BootloadHID should have been installed through qmk_install.sh, so the `:flash` method is really all that's needed.

Reply:
I think it's best to leave the 2nd option in to have a backup option as there's not much info on this particular board and it definitely works